.github/workflows/ci.yml

```diff
@@ -45,3 +45,6 @@ jobs:
 
       - name: Run Nim Light Client compiled with clang tests
         run: nix develop -c yarn test
+
+      - name: Run Nim groth16 verifier tests
+        run: nix develop -c yarn test-groth16-verifier
```

Comment:
We have a `Makefile`. We don't need to use `yarn` for non-JavaScript-related commands.

Reply:
Fixed